torch/csrc/jit/ir.cpp

```diff
@@ -569,6 +594,7 @@ void PythonOp::cloneFrom(Node * other_) {
     this->scalar_args.emplace_back(sa.get());
   }
 }
+#endif
```

Comment:
Can you put this at the top (in the excluded block)?

Reply:
Done